Tasks/KubernetesManifestV0/src/utils/DeploymentHelper.ts

```diff
@@ -78,21 +79,7 @@ function deployManifests(files: string[], kubectl: Kubectl, isCanaryDeploymentSt
 }
 
 async function checkManifestStability(kubectl: Kubectl, resources: Resource[]): Promise<void> {
-    const rolloutStatusResults = [];
-    const numberOfResources = resources.length;
-    for (let i = 0; i < numberOfResources; i++) {
-        const resource = resources[i];
-        if (models.workloadTypesWithRolloutStatus.indexOf(resource.type.toLowerCase()) >= 0) {
-            rolloutStatusResults.push(kubectl.checkRolloutStatus(resource.type, resource.name));
-        }
-        if (isEqual(resource.type, constants.KubernetesWorkload.pod, StringComparer.OrdinalIgnoreCase)) {
-            try {
-                await checkPodStatus(kubectl, resource.name);
-            } catch (ex) {
-                tl.warning(tl.loc('CouldNotDeterminePodStatus', JSON.stringify(ex)));
-            }
-        }
-    }
+    const rolloutStatusResults = await KubernetesManifestUtility.checkManifestStability(kubectl, resources);
```

Comment:
NIT: This can be moved to the place where this function is being invoked.

Reply:
Done.